subsys/emds/emds_flash_new.c

```diff
@@ -30,7 +24,7 @@ static void cand_list_init(sys_slist_t *cand_list, struct emds_snapshot_candidat
 	}
 }
 
-static void cand_put(sys_slist_t *cand_list, struct emds_snapshot_metadata *cand)
```

Comment:
bit lost, this is a brand new file added a few commits ago, why is it being fixed up in the same PR?